src/components/schemas/fields.yaml

```diff
@@ -691,3 +691,19 @@ bank_ifsc:
   description: >
       IFSC code of the bank account of the Employee
   pattern: '^[A-Z]{4}0[A-Z0-9]{6}$'
+
+is_implicitly_merged:
+  type: boolean
+  nullable: true
```

Comment:
let's remove the nullable and set it to False. Can populate the data once in prod and add not_null and set the default value to False